pkg/ceph/client/mon.go

```diff
@@ -72,20 +73,28 @@ func AppendAdminConnectionArgs(args []string, configDir, clusterName string) []s
 func ExecuteCephCommandPlain(context *clusterd.Context, clusterName string, args []string) ([]byte, error) {
```

Comment:
no, there's a difference between plain and no format.  no format means that the `--format` param isn't passed at all because it is invalid/illegal for the command being called.  the naming scheme with Plain and NoFormat is OK.